iot/api/CloudIotSample/Program.cs

```diff
@@ -1376,6 +1491,209 @@ public static object ClearRegistry(string projectId, string cloudRegion, string
         }
         //[END iot_clear_registry]
 
+
+        //[START iot_attach_device]
+        public static object AttachDevice(MqttClient client, string deviceId)
```

Comment:
When a user sees MqttClient passed in as an argument, they get frustrated and we'll see bug reports like "How do I create a MqttClient object?!?"  Every sample should include the code to create the client.

Reply:
Will do for every method